### ISSUE TITLE
Can I transform the input of INN into three-dimensional coordinates with an additional fixed fourth dimension?

### ISSUE THREAD

**Issue by ys830** [qianqianwang68/omnimotion#53]
Thank you very much for your amazing work!
I want to perform tracking on some three-dimensional data. Does the Invertible Neural Network (INN) support three-dimensional coordinate inputs? Do you have any suggestions?

**Comment by qianqianwang68** [comment]
yes, it supports three dimensional inputs. In fact the INN we used in our work takes exactly three dimensional coordinates.
